examples/src/main/java/io/grpc/examples/experimental/CompressingHelloWorldClient.java

```diff
@@ -43,6 +45,9 @@
   /** Construct client connecting to HelloWorld server at {@code host:port}. */
   public CompressingHelloWorldClient(String host, int port) {
     channel = ManagedChannelBuilder.forAddress(host, port)
+        .compressorRegistry(CompressorRegistry.getDefaultInstance())
+        .decompressorRegistry(DecompressorRegistry.getDefaultInstance())
+        .enableFullStreamDecompression()
```

Comment:
This is unstable and not fully implemented. We shouldn't have it in an example.

Reply:
Removed as the review comment